### ISSUE TITLE
ng g lib --route fails silently with @angular/cli 7.2.1

### ISSUE THREAD

**Issue by vincentpalita** [nrwl/nx#1001]
_[Please make sure you have read the submission guidelines before posting an issue](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#-submitting-issue)_

## Expected Behavior
Should generate a lib in libs module with Routing definitions.

## Current Behavior
The command fails silently, the lib is not generated.

## Failure Information (for bugs)
No error output, fails silently.

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Have the latest version of @angular/cli (7.2.1)
2. Run command: ng g lib my-lib --routing --directory=shared

### Context

- version of Nx used: 7.3.0
- version of Angular CLI used: 7.2.1
- angular.json:
```
{
  "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
  "version": 1,
  "newProjectRoot": "",
  "projects": {
    "my-app": {
      "root": "apps/my-app/",
      "sourceRoot": "apps/my-app/src",
      "projectType": "application",
      "prefix": "whoz",
      "schematics": {
        "@nrwl/schematics:component": {
          "styleext": "scss"
        }
      },
      "architect": {
        "build": {
          "builder": "@angular-devkit/build-angular:browser",
          "options": {
            "outputPath": "dist/apps/my-app/",
            "index": "apps/my-app/src/index.html",
            "main": "apps/my-app/src/main.ts",
            "polyfills": "apps/my-app/src/polyfills.ts",
            "tsConfig": "apps/my-app/tsconfig.app.json",
            "assets": [
              "apps/my-app/src/favicon.ico",
              "apps/my-app/src/assets"
            ],
            "styles": ["apps/my-app/src/styles.scss"],
            "scripts": []
          },
          "configurations": {
            "production": {
              "fileReplacements": [
                {
                  "replace": "apps/my-app/src/environments/environment.ts",
                  "with": "apps/my-app/src/environments/environment.prod.ts"
                }
              ],
              "optimization": true,
              "outputHashing": "all",
              "sourceMap": false,
              "extractCss": true,
              "namedChunks": false,
              "aot": true,
              "extractLicenses": true,
              "vendorChunk": false,
              "buildOptimizer": true,
              "budgets": [
                {
                  "type": "initial",
                  "maximumWarning": "2mb",
                  "maximumError": "5mb"
                }
              ]
            }
          }
        },
        "serve": {
          "builder": "@angular-devkit/build-angular:dev-server",
          "options": {
            "browserTarget": "my-app:build"
          },
          "configurations": {
            "production": {
              "browserTarget": "my-app:build:production"
            }
          }
        },
        "extract-i18n": {
          "builder": "@angular-devkit/build-angular:extract-i18n",
          "options": {
            "browserTarget": "my-app:build"
          }
        },
        "test": {
          "builder": "@angular-devkit/build-angular:karma",
          "options": {
            "main": "apps/my-app/src/test.ts",
            "polyfills": "apps/my-app/src/polyfills.ts",
            "tsConfig": "apps/my-app/tsconfig.spec.json",
            "karmaConfig": "apps/my-app/karma.conf.js",
            "styles": ["apps/my-app/src/styles.scss"],
            "scripts": [],
            "assets": [
              "apps/my-app/src/favicon.ico",
              "apps/my-app/src/assets"
            ]
          }
        },
        "lint": {
          "builder": "@angular-devkit/build-angular:tslint",
          "options": {
            "tsConfig": [
              "apps/my-app/tsconfig.app.json",
              "apps/my-app/tsconfig.spec.json"
            ],
            "exclude": ["**/node_modules/**"]
          }
        }
      }
    },
    "my-app-e2e": {
      "root": "apps/my-app-e2e",
      "projectType": "application",
      "prefix": "",
      "architect": {
        "e2e": {
          "builder": "@angular-devkit/build-angular:protractor",
          "options": {
            "protractorConfig": "apps/my-app-e2e/protractor.conf.js",
            "devServerTarget": "my-app:serve"
          },
          "configurations": {
            "production": {
              "devServerTarget": "my-app:serve:production"
            }
          }
        },
        "lint": {
          "builder": "@angular-devkit/build-angular:tslint",
          "options": {
            "tsConfig": "apps/my-app-e2e/tsconfig.e2e.json",
            "exclude": ["**/node_modules/**"]
          }
        }
      }
    }
  },
  "cli": {
    "warnings": {
      "typescriptMismatch": false,
      "versionMismatch": false
    },
    "defaultCollection": "@nrwl/schematics",
    "packageManager": "yarn"
  },
  "schematics": {
    "@nrwl/schematics:application": {
      "style": "scss"
    },
    "@nrwl/schematics:library": {
      "style": "scss"
    }
  },
  "defaultProject": "my-app"
}

```

- version of Angular DevKit used: 0.12.1
- 3rd-party libraries and their versions: nothing specific (tested in a blank project)

## Other
If I put the version "~7.1.0" for @angular/cli it works fine, the lib is generated (this actually setup the version 7.1.4 of @angular/cli).

With the version 7.2.1 of @angular/cli, without --routing the lib gets generated.

**Comment by pipozoft** [comment]
Any luck with this issue?

**Comment by macjohnny** [comment]
can't reproduce the issue with `@angular/cli@7.3.0` and `@nrwl/nx@7.5.1`, the library is generated correctly with `ng g lib my-lib --routing --directory=shared`

**Comment by Tre665** [comment]
Can confirm both. 
We had the same issue with cli 7.2 and nx 7.3.
But now with cli 7.2 and nx 7.5.1 it works again.

**Comment by vsavkin** [comment]
It looks like the issue has been fixed in Nx 7.5. If you can still reproduce the problem with the new version of Nx, could you reopen the issue with a repro?

Thanks!
